app/services/force/soql/preference_service.rb

```diff
@@ -7,11 +7,12 @@ class PreferenceService < Force::Base
       FIELD_NAME = :preferences
       FIELDS = load_fields(FIELD_NAME).freeze
 
-      def app_preferences_for_application(application_id)
+      def app_preferences_for_application(application_id, should_order: false)
         result = parsed_index_query(%(
           SELECT #{query_fields(:app_preferences_for_application)}
           FROM Application_Preference__c
           WHERE Application__c = '#{application_id}'
+          #{should_order ? 'Order by Preference_Order__c asc' : ''}
```

Comment:
Should `Order by` be capitalized to `ORDER BY`? Not sure what the convention is exactly

Reply:
SOQL states that their clauses are not case sensitive. But the rest of the code is using capital clauses so I will follow the same pattern.